<launch>

	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find localization_309512009)/config/rviz2.rviz" />

    <node name="tf_world" pkg="tf" type="static_transform_publisher" args="0 0 0 0 0 0 map world 10"/>

    <node name="tf_origin" pkg="tf" type="static_transform_publisher" args="1774 866 0 0 0 0 world origin 10"/>

	<node name="map_pub3" pkg="localization_309512009" type="map_pub3" output="screen"/>

	<node name="icp3" pkg="localization_309512009" type="icp3" output="screen"/>
	
	<arg name="path" default="/home/lct/nctu_sdc/localization_ws/" />
	<arg name="file" default="sdc_localization_3_lite" />
	<node pkg="rosbag" type="play" name="player" output="screen" args="-d 15 -r 0.03 --clock $(arg path)$(arg file).bag "/>

</launch>

